commonknowledge/meep-intelligence-hub
# SEO for hubs

Add DB columns and NextJS config for hub share cards.

## Description

- Adds favicon URL, SEO image to HubHomepage
- Use this data to render share card metadata in HTML (favicon URL, SEO image, SEO title, SEO description)

Closes CC-90, CC-66

## Motivation and Context

We want these microsites to be highly shareable and optimised for discovery algorithms.

## How Can It Be Tested?

1. Run migrations
2. Edit the HubHomepage in the wagtail admin, adding fields as above
3. If you want to, download [this chrome extension](https://chromewebstore.google.com/detail/social-share-preview/ggnikicjfklimmffbkhknndafpdlabib?hl=en) for previewing a page's social share card
4. Visit the hub homepage on localhost and inspect the social share card meta tags (or use the chrome extension to render it)
5. Page title should also be correct

## How Will This Be Deployed?

Nothing new.

## Screenshots (if appropriate):

<img width="183" alt="Screenshot 2024-05-30 at 21 55 32" src="https://github.com/commonknowledge/meep-intelligence-hub/assets/237556/9dde78f6-e3ea-4e33-afad-6cae6b54330a">

<img width="354" alt="Screenshot 2024-05-30 at 21 54 26" src="https://github.com/commonknowledge/meep-intelligence-hub/assets/237556/cfa1d940-426e-4896-822f-89f61312b860">

## Types of changes
